DEVX_ROADMAP.md

```diff
@@ -1,51 +1,39 @@
+* Adding `estimateTransactionComputationCost` for transaction cost estimation.
+* Adding `selectCoins` to select coins for gas payments, pay etc.
+* Standardizing the return type of u64 to string.
 
 ## SDK (Typescript, Rust)
-* Reliability improvements in Typescript SDK support for `sui_publish` API
-* Support for `sui_subscribeEvent` in the Typescript SDK
-* Rust SDK will be kept at parity with the Typescript SDK when feasible
-
-# Sui Move
-* Support for dynamic access of child objects to avoid explicitly passing child objects as transaction inputs and to enable implementation of design patterns/collections that require pointer-like parent->child representation. ([issue](https://github.com/MystenLabs/sui/issues/4203))
-* Support for passing strings and object ID encodings to entry functions
-* This frequently requested feature allows passing a vector of objects to entry functions which will open up a variety of flexible use cases *wrt* payments and coin management.
-* A collection of cryptographic libraries are exposed as Move native functions to use on-chain. They can be found at:
-  * [crypto.move](https://github.com/MystenLabs/sui/blob/4ed29873e35cad9d709dbd9370e17d261446e873/crates/sui-framework/sources/crypto/elliptic_curve.move): This contains various utility methods for on-chain signature verification. It also provides a method to recover Secp256k1 signature to its public key, bulletproof verification, and Pedersen commitments. See this work in action in the Cryptography (math) [example](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math).
-  * [elliptic_curve.move](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math): This introduces a RistrettoPoint struct, which represents an elliptic curve point on the Ristretto255 sub-group of curve25519. Currently, we support the addition and subtraction of elements in the Ristretto255 sub-group, and operations of scalars within the field of curve25519.
-  * HKDF, AES, ECDH
+
+* Intent signing support which will include an intent struct to be serialized and signed in addition to the transaction data [[Issues](https://github.com/MystenLabs/fastcrypto/issues/26)].
+* [Rust SDK] Removing reliance on String and Json Values in Rust Transaction Builder, making it more Rust friendly
+* Adding denomination conversion functionality, which converts SUI to MIST.
+* [TypeScript] Adding support for deserializing transactions.
+
+## Sui Move
+
+* Crypto.move will be split into its individual modules organized by crypto primitives [[PR](https://github.com/MystenLabs/sui/pull/4653)].
+* Supporting dynamic child access requires additional time due to newer complexities discovered and we are actively working on enabling it on devNet ASAP. 
+* Support for passing mutable coins instead of owned.
```

Comment:
Other items to mention
- Shared coins via `Safe` module: https://github.com/MystenLabs/sui/pull/4680
- Support for type reflection: https://github.com/move-language/move/pull/566
- Support for `_`'s in integer literals (e.g., `let one_million = 1_000_000`)